pkgs/build-support/replace-secret/replace-secret.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3 }:
```

Comment:
Perhaps add a check phase here that does some trivial usage of the command so we can be sure it still works without having to run nixos tests.

Reply:
Yeah, that's a good idea. I added two simple checks. If the diff looks okay to you, I'll squash it.